css/index.css

```diff
@@ -1,3 +1,326 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
 
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* index */
+.navigation a {
+	text-decoration: none;
+	color: lightgray;
+	padding: 30px;
+}
+
+.head-nav {
+	width: 1000px;
+    margin: auto;
+    padding-bottom: 75px;
+	display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.intro {
+    width: 1000px;
+    padding-bottom: 75px;
+	margin: auto;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.intro div {
+	text-align: center;
+	padding: 50px;
+}
+
+.intro h1 {
+	font-size: 72px;
+}
+
+.intro h6 {
+    border: 1px solid black;
+    width: 175px;
+    padding: 8px;
+    margin: auto;
+}
+
+.blurb-one {
+	width: 1000px;
+	margin: auto;
+    display: flex;
+    justify-content: center;
+}
+
+.blurb-one div {
+	width: 49%;
+	padding: 10px;
+}
+
+.middle-img {
+	margin: auto;
+	padding: 25px;
+	display: flex;
+}
+
+.blurb-two {
+    width: 1000px;
+    padding-bottom: 50px;
+	margin: auto;
+    display: flex;
+    justify-content: center;
+	
+}
+
+.blurb-two div {
+	width: 30%;
+	padding: 10px;
+}
+
+.contact {
+	padding: 10px;
+	width: 1000px;
+	margin: auto;
+}
+
+footer h6 {
+	text-align: center;
+}
+
+hr {
+    width: 900px;
```

Comment:
Having an absolute width on your hr causes it to overlap the content at some viewport widths, and to be too small at others. Try to use a percentage, or even use a border on an HTML element instead.